src/MockPlayerBridge.ts

```diff
@@ -41,6 +76,85 @@ export class MockPlayerBridge extends AbstractPlayerBridge {
     return 'dummy-stream-id';
   }
 
+  protected pause() {
+    this.playbackState = 'PAUSED';
+    document.dispatchEvent(new Event('livery-playback-change'));
```

Comment:
The bridge should not dispatch events; it's the job of the player.

This is also not a proper livery-playback-change LiveryPlaybackChangeEvent instance which is confusing to me at least.

For the purpose of having the LiveryBridgeInteractive deal with playbackState changes that should just use the subscribePlaybackState() method and it's listener.

I suppose for mocking in here you could use some kind of event dispatching, but then I'd prefer to just see an array of playbackListeners being used from within subscribePlaybackState() and a private setPlaybackState() method here.

The same goes for the events dispatched below.